package.json

```diff
@@ -212,5 +212,7 @@
   "pre-commit": [
     "lint-staged"
   ],
-  "sideEffects": false
+  "sideEffects": [
+    "es/**/style/*"
```

Comment:
what about styles under `lib/` and `dist/`?

Reply:
This problem only exist when use tree shaking with es6 module, so not need to include `lib` ant `dist`.

Reply:
`babel-import-plugin` imports from `lib` folder so there's still no css bundled in 3.4.3

Reply:
let me see, @whtsky do you have any repo for test.

Reply:
I don't have any public repo for this, but I can confirm there's still no css in our project with
`['import', { libraryName: 'antd', style: true }]` config.

I have already opened #10219 for this